src/pages/Home.jsx

```diff
@@ -1,17 +1,71 @@
-import { Button, ThemeProvider } from '@material-ui/core';
 import { useState } from 'react';
 import { useHistory } from 'react-router-dom';
 import Typography from '@material-ui/core/Typography';
+import { Button, ThemeProvider } from '@material-ui/core';
 
 import OTPInput from '../components/otp-input';
 import './Home.css';
 import { createMuiTheme } from '@material-ui/core/styles';
+import axios from 'axios';
 
-const Home = () => {
-  const [joiningRoom, setJoiningRoom] = useState(false);
+const scopes = [
+  'ugc-image-upload',
+  'user-read-playback-state',
+  'user-modify-playback-state',
+  'user-read-currently-playing',
+  'streaming',
+  'app-remote-control',
+  'user-read-email',
+  'user-read-private',
+  'playlist-read-collaborative',
+  'playlist-modify-public',
+  'playlist-read-private',
+  'playlist-modify-private',
+  'user-library-modify',
+  'user-library-read',
+  'user-top-read',
+  'user-read-playback-position',
+  'user-read-recently-played',
+  'user-follow-read',
+  'user-follow-modify',
+];
+
+const state = '34fFs29kd09';
 
+const clientId = '88616603e64046f4a64cec1b94f02e1c';
+
+const clientSecret = '291c5fef94874c789fd0927afc4a652e';
```

Comment:
we should probably hold the client secrets in a `.env` file and also place it in a `.gitignore` so that we don't expose "secrets"